### PR TITLE
Feature/spark

### PR DESCRIPTION
**Pull request by annaoliinyk**

### CODE REVIEW

File: pyspark_analysis/pyspark_tasks.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                        StructField("origin_country", StringType(), True),
+                        StructField("time_position", StringType(), True),
+                        StructField("last_contact", StringType(), True), StructField("longitude", StringType(), True),
+                        StructField("latitude", StringType(), True), StructField("baro_altitude", StringType(), True),
+                        StructField("on_ground", StringType(), True), StructField("velocity", StringType(), True),
+                        StructField("true_track", StringType(), True), StructField("vertical_rate", StringType(), True),
+                        StructField("sensors", ArrayType(IntegerType()), True),
+                        StructField("geo_altitude", StringType(), True), StructField("squawk", StringType(), True),
+                        StructField("spi", StringType(), True), StructField("position_source", StringType(), True)])
+france_neighbours = ["Belgium", "Luxembourg", "Germany", "Switzerland", "Italy", "Monaco", "Spain", "Andorra"]
```
Comment (AnaKovaGH) [inline pyspark_analysis/pyspark_tasks.py]: Move to configs

File: pyspark_analysis/pyspark_tasks.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+
+def create_df():
+    json_as_df = SPARK.read.json(JSON_PATH)
+    states_only_column = json_as_df.select("states").collect()[0]
+    states_list = states_only_column.__getitem__("states")
```
Comment (AnaKovaGH) [inline pyspark_analysis/pyspark_tasks.py]: Look how you can change __getitem__

File: pyspark_analysis/pyspark_tasks.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    max_altitude_df2 = df.select(df["icao24"], df["geo_altitude"]).filter(df["geo_altitude"] == max_altitude)
+    show_df(max_altitude_df2)
+    return max_altitude_df2
+
+
+def get_highest_velocity(df: DataFrame):
+    print("\nAirplane(s) with highest velocity:")
+    max_velocity_df = df.select([max("velocity")])
+    max_velocity = max_velocity_df.first()[0]
+    max_velocity_df2 = df.select(df["icao24"], df["velocity"]).filter(df["velocity"] == max_velocity)
```
Comment (AnaKovaGH) [inline pyspark_analysis/pyspark_tasks.py]: Change for more meaningful name

File: pyspark_analysis/pyspark_tasks.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    show_df(max_velocity_df2)
+    return max_velocity_df2
+
+
+def get_airplanes_count_by_airport(df: DataFrame):
+    print("\nCount of airplanes by airport:")
```
Comment (AnaKovaGH) [inline pyspark_analysis/pyspark_tasks.py]: Change for logging

File: pyspark_analysis/pyspark_tasks.py [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    airplanes_count_df = df.groupBy("origin_country").count()
```
Comment (AnaKovaGH) [inline pyspark_analysis/pyspark_tasks.py]: Move origin_country to a config file

File: pyspark_analysis/pyspark_tasks.py [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    get_country_min_count_airplanes(airplanes_count_df)
+    get_country_max_count_airplanes(airplanes_count_df)
+    get_count_for_countries_on_c(airplanes_count_df)
+    get_count_for_countries_on_g(airplanes_count_df)
+    get_count_for_france_neighbours(airplanes_count_df)
+    return airplanes_count_df
+
+
+def get_country_min_count_airplanes(airplanes_df: DataFrame):
+    print("\nCountries with smallest number of airplanes")
```
Comment (AnaKovaGH) [inline pyspark_analysis/pyspark_tasks.py]: The same - use logging

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please look at the comments

**Comment by annaoliinyk** [comment]
This was fixed,
Ready for review

**Review by AnaKovaGH — APPROVED** [review]
LGTM

**Pull request merged**